docs/insights/api.md

```diff
@@ -0,0 +1,511 @@
+| Links     | Links to other resources.  See [REST API Links](https://octopus.com/docs/octopus-rest-api#api-links) |
+
+## Update an Insights report
+PUT: api/{spaceId}/insights/reports/{reportId}
+Updates an existing Insights report.
+
+### Parameters
+| Name | Description |
+| ---- | ----------- |
+| spaceId  | Id of the space containing the report |
+| reportId | The id of the Insights report to update |
+
+### Request Body
+The request body is in JSON format.
+
+### Sample request
+```json
+{
+    "SpaceId": "Spaces-1",
+    "Id": "InsightsReports-1",
+    "Name": "High Volume Report",
+    "Description": "High volume tenanted deployments report",
+    "ProjectGroupIds": ["ProjectGroups-1", "ProjectGroups-2"],
+    "ProjectIds": ["Projects-1", "Projects-2"],
+    "TenantIds": ["Tenants-1", "Tenants-2", "Tenants-3"],
+    "TenantTags": [
+      "Deployment Group/Group A",
+      "Deployment Group/Group B",
+      "Deployment Group/Group C",
+      "Location/Australia",
+      "Location/Asia",
+      "Location/Europe",
+      "Location/US"
+    ],
+    "TenantMode": "TenantedAndUntenanted",
+    "EnvironmentGroups": [
+      {
+        "Name": "Test",
+        "Environments": ["Environments-1", "Environments-2"]
+      },
+      {
+        "Name": "Internal",
+        "Environments": ["Environments-3", "Environments-4"]
+      },
+      {
+        "Name": "Production",
+        "Environments": ["Environments-5", "Environments-6"]
+      }
+    ],
+    "TimeZone": "UTC"
+}
+```
+
+### Properties
+| Property | Required | Description |
+| -------- | -------- | ----------- |
+| SpaceId  | Required | The id of the space containing the report. | 
+| Id       | Required | The id of the report to be updated. | 
+| Name     | Required | The name of the report. | 
+| Description | Optional | The description of the report. | 
+| ProjectGroupIds | Optional | An array of the project group ids to be included in the report. | 
+| ProjectIds | Optional | An array of the project ids to be included in the report. | 
+| TenantIds  | Optional | An array of the tenant ids to be  included in the report. | 
+| TenantTags | Optional | An array of the tenant tags  to be associated with this report. | 
+| TenantMode | Optional | Indicates what types of deployments will be contributing to this report. May be TenantedAndUntenanted, Tenanted, or Untenanted. | 
+| EnvironmentGroups | Optional | An array of the environment groups to be  included in this report. | 
+| TimeZone | Optional | The timezone to be used in the report. Must be either a Windows or IANA (Tzdb) timezone. | 
+
+## Delete an Insights report
+DELETE: api/{spaceId}/insights/reports/{reportId}
+Deletes an existing Insights report.
+
+### Parameters
+| Name | Description |
+| ---- | ----------- |
+| spaceId  | Id of the space containing the report |
+| reportId | The id of the Insights report to delete |
+
+## Get deployment level insights metrics
+GET: api/{spaceId}/insights/reports/{reportId}/deployments
```

Comment:
I don't know if the leading slash should be added. I checked my API, and it included one
```suggestion
GET: `/api/{spaceId}/insights/reports/{reportId}/deployments`
```